src/generators/app/index.js

```diff
@@ -45,6 +42,23 @@ module.exports = generator.Base.extend({
       camelCaseName: _.camelCase(this.appname)
     };
 
-    this.fs.copyTpl([this.templatePath('**/*'), this.templatePath('**/.*')], '', vars);
-  }
+    var input = [this.templatePath('**/*'), this.templatePath('**/.*')];
+    return this.fs.copyTpl(input, '', vars);
+  },
+
+  initGitRepo: function () {
+    this.composeWith(
+      'git-init',
+      {
+        options: { commit: `Initialize Kibana Plugin (${pkg.name} v${pkg.version})` }
+      },
+      {
+        local: require.resolve('generator-git-init'),
+      }
+    );
+  },
```

Comment:
I wanted it so that the plugin had a base, and so that history was sort of established. I also wanted to track which version of the generator created the output. It's super seamless, but maybe it's out of line. What do others think?